http-server/src/main/java/org/triplea/server/error/reporting/ErrorReportController.java

```diff
@@ -0,0 +1,52 @@
+package org.triplea.server.error.reporting;
```

Comment:
This is technically not a new file, as we had a controller previously: https://github.com/triplea-game/triplea/pull/4846/files#diff-7e7834725765093021ce16251828b07c

Given drop-wizard will do the JSON serialization and deserialization for us automatically, it is a bit simpler. On the other hand the check for empty fields so we throw an illegal arg adds complexity as does the new way to grab IP address from the request.

Side-note, I'm pretty sure we could use javax validator annotations to do the same check as we do here for empty fields. For now keeping things simple and trying to focus on getting the http server up and running and functional.